docs/en/02_Developer_Guides/01_Templates/03_Requirements.md

```diff
@@ -240,6 +240,10 @@ You can also use any of the below options in order to tweak this behaviour:
    other systems, it is usually necessary to distinguish combined files from other assets.
  * `Requirements_Backend.combine_in_dev` - By default combined files will not be combined except in test
    or live environments. Turning this on will allow for pre-combining of files in development mode.
+ * `Requirements_Backend.resolve_relative_css_refs` - By default combined files will be parsed for relative urls 
+   to image/font assets and those utls will be changed to absolute to accomodate to the fact that the combined 
```

Comment:
```suggestion
   to image/font assets and those URLs will be changed to absolute to accommodate the fact that the combined 
```

On top of that, I'm not sure this should be turned off by default, but we can discuss that separately.

Reply:
Thanks for the spellcheck :)
Wait, do you mean " not sure this should be turned OFF by default" or "turned ON"?